Validation/EcalDigis/interface/EcalSelectiveReadoutValidation.h

```diff
@@ -7,47 +7,32 @@
  *
  */
 
-#include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDAnalyzer.h"
-#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "DQMServices/Core/interface/DQMEDAnalyzer.h"
+
+#include "DQMServices/Core/interface/MonitorElement.h"
 
-#include "DataFormats/Common/interface/Handle.h"
 #include "DataFormats/EcalDigi/interface/EcalDigiCollections.h"
 #include "SimDataFormats/CaloHit/interface/PCaloHit.h"
 
-
-#include "Geometry/CaloTopology/interface/EcalTrigTowerConstituentsMap.h"
-#include "DataFormats/EcalDetId/interface/EcalScDetId.h"
-#include "DataFormats/EcalDetId/interface/EcalTrigTowerDetId.h"
-
 #include "DataFormats/EcalDigi/interface/EcalDigiCollections.h"
-//#include "DataFormats/EcalDetId/interface/EEDetId.h"
-//#include "DataFormats/EcalDetId/interface/EBDetId.h"
 #include "DataFormats/EcalRecHit/interface/EcalRecHitCollections.h"
-#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
 #include "DataFormats/FEDRawData/interface/FEDRawDataCollection.h"
+#include "DataFormats/EcalDetId/interface/EcalScDetId.h"
+#include "DataFormats/EcalDetId/interface/EcalTrigTowerDetId.h"
 
 #include "Validation/EcalDigis/src/CollHandle.h"
 
 #include <string>
 #include <set>
-#include <utility>
 #include <fstream>
-#include <inttypes.h>
-#include "DQMServices/Core/interface/DQMStore.h"
-#include "DQMServices/Core/interface/MonitorElement.h"
 
-class DQMStore;
-class MonitorElement;
 class EBDetId;
 class EEDetId;
-class EcalTPParameters;
 class EcalElectronicsMapping;
+class EcalTrigTowerConstituentsMap;
 
-class EcalSelectiveReadoutValidation: public edm::EDAnalyzer{
+class EcalSelectiveReadoutValidation: public DQMEDAnalyzer{
```

Comment:
The static analyzer report for 'modules to thread unsafe globals' reports this module and therefore it can not be a DQMEDAnalyzer 
```
In call stack '  EcalSelectiveReadoutValidation::analyze() calls function  EcalSelectiveReadoutValidation::analyzeEE() calls function  EcalSelectiveReadoutValidation::frame2Energy() const static variable  EcalSelectiveReadoutValidation::frame2Energy() const::firstCall' is accessed , 'EcalSelectiveReadoutValidation::analyze()' overrides 'edm::EDAnalyzer::analyze() virtual'

In call stack '  EcalSelectiveReadoutValidation::analyze() calls function  EcalSelectiveReadoutValidation::compareSrfColl<const class edm::SortedCollection<class EESrFlag, struct edm::StrictWeakOrdering<class EESrFlag> >>() calls function  EcalSrFlag::flagName() const calls function  EcalSrFlag::flagName() static variable  EcalSrFlag::srfNames' is accessed , 'EcalSelectiveReadoutValidation::analyze()' overrides 'edm::EDAnalyzer::analyze() virtual'
```